doc/docusaurus/docs/glossary.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 15
+---
+
+# Glossary
+
+### Address
+
+Each UTXO has an address, which stipulates the conditions for spending the UTXO.
+
+An address on Cardano can be based on either a public key hash, which requires a private key signature corresponding to the public key hash to spend the UTXO, or a script hash, which requires the script with that particular hash to be executed to spend the UTXO.
+These are referred to as public key address and script address, respectively.
+
+### Cardano
+
+The blockchain system for which Plutus Core and Plutus Tx are built.
+
+### Contract Blueprint
+
+A contract blueprint enables communication between on-chain code and off-chain code written in different languages.
+It is introduced in [CIP-57](https://developers.cardano.org/docs/governance/cardano-improvement-proposals/cip-0057/).
+Also see [Producing a Plutus Contract Blueprint](./working-with-scripts/producing-a-blueprint.md).
+
+### Currency Symbol and Token Name
+
+On Cardano, each class of tokens (or asset class) is identified by its currency symbol (also known as currency, or asset group) along with the token name.
+The minting and burning of a token are controlled by the Plutus script whose hash corresponds to the token's currency symbol.
+
+Ada/Lovelace is a special asset class where the currency symbol and token name are both empty strings, and it cannot be minted or burned via transactions.
```

Comment:
Isn't `Ada` the actual currency, while `Lovelace` is just a unit to describe it?

Reply:
If anything it's the other way around: the unit used everywhere is Lovelace, and Ada is for convenience.